### PR TITLE
Some refactoring of inference_run.py and tools.py

### PR DESCRIPTION
**Pull request by shubhra**
-Separated out load model functionality into its own method
-Separated out load seeds file functionality into its own method
-Added support for loading manual seeds text file
-Improved docstrings, removed unused imports & code, added type hints

### CODE REVIEW

File: core/tool/tools.py [hunk 1]
```diff
@@ -1,186 +1,103 @@
...
+                seeds = list(seeds)
+                seeds = {i + 1: coord for i, coord in enumerate(seeds)}
+        else:
+            # Expects a .txt file of seed coordinates in a Dict format
+            # Each line as {id: [z, y, x]}
+            with open(seeds_file_path, 'r') as f:
+                contents = f.read()
+                seeds = ast.literal_eval(contents)
+    else:
+        print('{} - seeds file does not exist!'.format(seeds_file_path))
```
Comment (jmrk84) [inline core/tool/tools.py]: I suggest generally using f strings instead, see https://www.python.org/dev/peps/pep-0498/

File: core/tool/tools.py [hunk 2]
```diff
@@ -1,186 +1,103 @@
...
+def resume_dict_load(dict_path: str, name: str, resume_obj: int) -> Dict:
+    """Load dict to resume processing."""
+    dict_path = os.path.join(dict_path, name, '.pkl')
+    if os.path.exists(dict_path):
+        resume = load_obj(dict_path)
     else:
         resume = {'resume_seed': resume_obj}
     resume_obj = resume['resume_seed']
-    print('resume', resume_obj)
+    print('Resume', resume_obj)
```
Comment (jmrk84) [inline core/tool/tools.py]: f-string here as well.

File: core/tool/tools.py [hunk 3]
```diff
@@ -1,186 +1,103 @@
...
+            with h5py.File(seeds_file_path, 'r') as f:
+                seeds = f['seeds'][()]
+                seeds = list(seeds)
+                seeds = {i + 1: coord for i, coord in enumerate(seeds)}
+        else:
+            # Expects a .txt file of seed coordinates in a Dict format
+            # Each line as {id: [z, y, x]}
+            with open(seeds_file_path, 'r') as f:
+                contents = f.read()
+                seeds = ast.literal_eval(contents)
```
Comment (jmrk84) [inline core/tool/tools.py:86]: Not really important, but using ast.literal_eval to parse simple data like this seems like a non-standard way to serialize things. I would have used one seed per line with id: z, y, x with "manual line parsing" or json / YAML.

**Review by jmrk84 — CHANGES_REQUESTED** [review]

File: core/tool/tools.py (hunk 3 above)
Comment (shubhra) [reply core/tool/tools.py:86]: I'll add this as a part of a later change if we decide to go for this.

File: core/tool/tools.py (hunk 2 above)
Comment (shubhra) [reply core/tool/tools.py]: Updated.

File: core/tool/tools.py (hunk 1 above)
Comment (shubhra) [reply core/tool/tools.py]: Updated.

**Pull request merged**